forumdirectory/forumdirectory.php

```diff
@@ -11,6 +11,12 @@
 use Friendica\Content\Widget;
 use Friendica\Core\Config;
 use Friendica\Database\DBM;
+use Friendica\Model\Profile;
+
+require_once 'boot.php';
+require_once 'include/dba.php';
+require_once 'include/plugin.php';
+require_once 'include/text.php';
```

Comment:
Are all these new includes needed?

Reply:
Yes they are, there's no autoload for includes, so we were relying on the fact that these files would be included *at some point before* in the execution chain, but it isn't safe.

Reply:
Ah, okay. This really confused me a lot.